### PR TITLE
Adds dates to logging

### PR DESCRIPTION
**Pull request by obiwen**
- alters logback output to include the current date
- alters full json logging to include a current date

### CODE REVIEW

File: com.swookiee.runtime.core/src/main/java/com/swookiee/runtime/core/internal/logging/FullJsonLayout.java [hunk 1]
```diff
@@ -23,6 +26,10 @@
         @SuppressWarnings("unchecked")
         Map<String, Object> jsonMap = super.toJsonMap(event);
 
+        DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss:SSS'Z'");
+        Calendar cal = Calendar.getInstance();
```
Comment (larsp) [inline com.swookiee.runtime.core/src/main/java/com/swookiee/runtime/core/internal/logging/FullJsonLayout.java]: should have been declared outside of the method

**Comment by larsp** [comment]
For the Java portion of it I would have prefered using jacksons util class `ISO8601DateFormat`:

```
...
    private static final ISO8601DateFormat ISO_FORMATTER = new ISO8601DateFormat();

    @Override
    protected Map<String, Object> toJsonMap(final ILoggingEvent event) {
        @SuppressWarnings("unchecked")
        Map<String, Object> jsonMap = super.toJsonMap(event);
        jsonMap.put("ts", ISO_FORMATTER.format(new Date()));
...
```

**Comment by obiwen** [comment]
Using the Jackson ISO8601DateFormat would introduce the dependency to the Jackson package within this module. I was under the impression, that this should be avoided within the core module.

**Comment by larsp** [comment]
It already has since the `ch.qos.logback.contrib` package needs it and is used already in the `core`:

```
 ch.qos.logback.contrib.jackson,
 ch.qos.logback.contrib.json,
 ch.qos.logback.contrib.json.classic,
```

**Comment by larsp** [comment]
The title is a little misleading but will be merged :)

**Pull request merged**